frontend/src/dashboard/panels/EventHistory.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import useAxios from 'axios-hooks';
+import {
+  Box,
+  CircularProgress,
+  Paper,
+  Typography
+} from '@material-ui/core';
+
+import { Event } from '../../types/interfaces';
+
+const mockEvents: Event[] = [
+  {
+    event_id: 4, 
+    host_id: 1,
+    location: '100 Union Dr',
+    event_date: new Date(),
+    is_private: false,
+    event_summary: "PAX East",
+    addtional_links: 'a',
+  },
+  {
+    event_id: 5, 
+    host_id: 2,
+    location: 'Prospect Park',
+    event_date: new Date(),
+    is_private: false,
+    event_summary: "Llamapalooza",
+    addtional_links: 'a',
+  },
+]
+
+const EventHistory = () => {
+    const [{ data, loading, error }] = useAxios<[Event]>({
+      url: `/api/user/events/past`,
+    }, { manual: false, useCache: false });
+
+    return (
+      loading 
+        ? <CircularProgress color="secondary"/>
+        : (
+            !error
+              ? <Typography variant="body1">There was an error: {JSON.stringify(error)}</Typography>
+              : <>
+                  {mockEvents.map((event) => (
+                    <Paper elevation={2} style={{marginBottom: '16px'}} key={event.event_id}>
+                      <Box p={1}>
+                        <Typography variant="body1">{event.event_summary}</Typography>
+                      </Box>
+                    </Paper>
+                  ))}
+                </>
+        )
+    );
```

Comment:
I haven't tested this code yet, but this reminded me: how long does it typically take to start up a React App from the command line? Reloads tend to be quick enough but that first server startup can take upwards of minutes for me. Maybe it's my shitty laptop, maybe it's normal for devving a React app, but I wanted to ask anyway

Reply:
For me, on my laptop that I got freshman year the first startup is definitely significantly slower than on my desktop, but doesn't usually take more than about 30 seconds to a minute. That's really weird that it takes so long on your machine.